Week1/homework/app.js

```diff
@@ -1,11 +1,132 @@
 'use strict';
 
 {
+  // 1.1
   const bookTitles = [
-    // Replace with your own book titles
-    'harry_potter_chamber_secrets'
+    'the_psychopath_test',
+    'against_happiness',
+    'the_bell_jar',
+    'american_psycho',
+    'amsterdam',
+    'the_brief_history_of_the_dead',
+    'a_clockwork_orange',
+    'columbine',
+    'on_seeing_and_noticing',
+    'the_craftsman'
   ];
+  // 1.2
+  // 1.3
+  function createList() {
+    const ul = document.createElement('ul');
+    bookTitles.forEach(title => {
+      const li = document.createElement('li');
+      li.textContent = title;
+      ul.appendChild(li);
+    });
+    document.body.appendChild(ul);
+  }
+  window.onload = createList;
```

Comment:
You can assign only one event handler function to `window.onload`. Since you assign it in three different places in your code, the last one (in line 131) wins. Consequently your  `createList` and `displayBooksInfo()` functions are never called.